File: python/reverse_words_in_a_string_iii.test.py
import unittest
import reverse_words_in_a_string_iii

class Test(unittest.TestCase):
  def test_1(self):
    s = "Let's take LeetCode contest"
    result = reverse_words_in_a_string_iii.Solution().reverseWords(s)
    self.assertEquals("s'teL ekat edoCteeL tsetnoc", result)

unittest.main(verbosity=2)

File: java/src/test/java/com/example/leetcode/ReverseWordsInAStringIIITest.java
package com.example.leetcode;

import org.junit.Test;

import static org.junit.Assert.*;

public class ReverseWordsInAStringIIITest {

    private ReverseWordsInAStringIII solution = new ReverseWordsInAStringIII();

    @Test
    public void test() {
        String s = "Let's take LeetCode contest";

        String result = solution.reverseWords(s);
        assertEquals("s'teL ekat edoCteeL tsetnoc", result);
    }
}